packages/db/bin/server.js

```diff
@@ -1,14 +1,8 @@
-const Config = require("@truffle/config");
-
 const { serve } = require("@truffle/db");
 
 const port = 4444;
 
-const config = Config.detect({
-  workingDirectory: process.argv[2] || process.cwd()
-});
-
-const server = serve(config);
+const server = serve();
```

Comment:
Should we still pass the config info here? @fainashalts might be using that functionality.

Reply:
So what was the result of our short conversation last week about this? I'm not sure we're using anything from `config`?

Reply:
We might want to still allow passing in that path, for the use case of running `yarn start <dir>` with a truffle-config that overrides the default.

Reply:
Keeping the `Config.detect`!